src/execution_plan/ops/op.h

```diff
@@ -91,7 +91,9 @@ typedef struct OpBase OpBase;
 
 void OpBase_Init(OpBase *op);       // Initialize op.
 void OpBase_Free(OpBase *op);       // Free op.
-void OpBase_Reset(OpBase *op);      // Reset op.
 Record OpBase_Consume(OpBase *op);  // Consume op.
 Record OpBase_Profile(OpBase *op);  // Profile op.
 int OpBase_ToString(const OpBase *op, char *buff, uint buff_len);
+
+void OpBase_PropegateFree(OpBase *op); // Sends free request to each operation up the chain.
+void OpBase_PropegateReset(OpBase *op); // Sends reset request to each operation up the chain.
```

Comment:
(Kind of a nit, but these are important calls) Typo, these should be `OpBase_PropagateReset` and `OpBase_PropagateFree`

Reply:
👍